src/views/Timeline.vue

```diff
@@ -40,68 +40,87 @@
           ]"
         >
           <UiButton class="pr-3">
-            {{ $t(`proposals.states.${state}`) }}
+            {{ $t(`proposals.states.${filterBy}`) }}
             <Icon size="14" name="arrow-down" class="mt-1 mr-1" />
           </UiButton>
         </UiDropdown>
       </div>
+
       <Block v-if="spaces.length < 1 && !scope" class="text-center">
         <div class="mb-3">{{ $t('noFavorites') }}</div>
         <router-link :to="{ name: 'home' }">
           <UiButton>{{ $t('addFavorites') }}</UiButton>
         </router-link>
       </Block>
+
       <Block v-else-if="loading" :slim="true">
         <RowLoading class="my-2" />
       </Block>
 
-      <NoResults
-        :block="true"
-        v-else-if="Object.keys(this.proposals).length < 1"
-      />
+      <NoResults :block="true" v-else-if="proposals.length < 1" />
       <div v-else>
         <Block :slim="true" v-for="(proposal, i) in proposals" :key="i">
           <TimelineProposal :proposal="proposal" :i="i" />
         </Block>
       </div>
+      <div id="endofpage" />
+      <Block v-if="loadingMore && !loading" :slim="true">
```

Comment:
You could remove this code https://github.com/snapshot-labs/snapshot/pull/323/files#diff-49c24724cfee2d52d020a60bfcc67936946b2e597c80639662c20ec405d26814R56 and use this RowLoading for both first load page and new page

Reply:
Yeah, I wasn't sure how that could be done was we have a lot of other condition before and after.

Reply:
I feel like they kind of need to be in that order to work.

Reply:
Ok yes, it's fine like this